spec/lib/tracked_revisions_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_dependency "#{Rails.root}/lib/analytics/course_statistics.rb"
+
+describe 'TrackedRevisions' do
+  let(:course) { create(:course, start: '30-05-2019'.to_date, end: '31-07-2019'.to_date) }
+  let(:user) { create(:user, username: 'Textorus') }
+
+  before do
+    course.students << user
+  end
+
+  it 'fetches the course stats for only tracked articles' do
+    VCR.use_cassette 'course_upload_importer/Textorus' do
+      UpdateCourseStats.new(course)
+    end
+    subject = CourseStatistics.new([course.id]).report_statistics
+    expect(subject[:articles_edited]).to eq(151)
```

Comment:
This is too specific for such a large number of articles; if there are pages or edits that get deleted sometime in the future, the spec will break.

I think exact numbers are okay if there are far fewer edits involved, though. I would suggest an example user and time range where there are just a few articles edited. Here's perhaps a good one (from one our our courses last summer): https://en.wikipedia.org/wiki/Special:Contributions/Hbultra

If you use the article title find find the right article and ArticlesCourses records to untrack, the test will be very easy to understand (and to compare expected results by viewing the contributions history).